go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/omec-project/upf-epc
```

Comment:
Is there a reason why this file was moved to the repo root? Is it to later facilitate pfcp-agent extraction in its own repo?

Reply:
`go test` requires `go.mod` to run, and it was easier to move it to the root repo. The fact that it facilitates the repo extraction is a side-effect.